src/ray/core_worker/common.h

```diff
@@ -122,6 +133,9 @@ struct ActorCreationOptions {
   /// When true, the child task will always scheduled on the same placement group
   /// specified in the PlacementOptions.
   bool placement_group_capture_child_tasks = true;
+  /// Environment variables to update for this actor.  Maps a variable name to its
+  /// value.  Can override existing environment variables and introduce new ones.
```

Comment:
Consider adding a comment that these will be propagated to any child actors and/or tasks.